Magento2/Tests/Legacy/ClassesPHPUnitTest.php

```diff
@@ -0,0 +1,30 @@
+<?php
+/**
+ * Copyright © Magento. All rights reserved.
+ * See COPYING.txt for license details.
+ */
+namespace Magento2\Tests\Legacy;
+
+use PHP_CodeSniffer\Tests\Standards\AbstractSniffUnitTest;
+
+class ClassesPHPUnitTest extends AbstractSniffUnitTest
+{
+    /**
+     * @inheritdoc
+     */
+    public function getErrorList()
+    {
+        return [
+            7 => 1,
+            9 => 1,
```

Comment:
Is the expression on line 8 and 10 in the test file perfectly valid in the context of Magento **2**?

Reply:
@sivaschenko what are perfectly valid expression in the context of Magento 2? I've made up those examples as an interpretation of the regular expression, but as you said this may be no longer accurate. Sniff in particular refers to PSR1 https://www.php-fig.org/psr/psr-1.

Reply:
I believe we should simplify the test to fail in case of any static calls to these methods independently from the passed parameter. Also, I'd fail the test if `Mage` class is mentioned